.travis.yml

```diff
@@ -13,16 +14,17 @@ rvm:
   - rbx-3
 env:
   global:
-    - JRUBY_OPTS='--debug' # get more accurate coverage data in JRuby
+    - JRUBY_OPTS='--debug -J-Xmx1000M' # get more accurate coverage data in JRuby
   matrix:
     - 'TASK=spec'
     - 'TASK=ascii_spec'
     - 'TASK=internal_investigation'
 matrix:
   allow_failures:
-    - rvm: jruby-9.0.1.0
+    - rvm: jruby-9.1.13.0
```

Comment:
Why? `jruby-9.0.1.0` was temporarily moved to `allow_failures` because of a CI problem unrelated to RuboCop. But it's working now and I think RuboCop still officially supports jruby by making sure it's build is green...

Reply:
Because `jruby-9.0.1.0` is not in the list of rubies that are actually run on the CI.

Reply:
I mean the "plus" part of the diff, not the removal. Why putting `jruby-9.1.13.0` in allowed failures?

Reply:
Same answer. 😛 `9.0.1.0` was replaced with `9.1.13.0` in the list of rubies (L:6).

Reply:
Oh. I see what you mean now @deivid-rodriguez. Got hung up on the number. Yeah. Should be passing now?

Reply:
I don't get it. We were enforcing jruby to pass when using `9.0.1.0`, then we _temporarily_ removed it because of a CI issue, now we upgrade to `9.1.13.0` and things are working in CI. Shouldn't it be again not present in `allowed_failures`?

I understand if you want to split that change to a separate commit, I guess. But the final diff here should not include that in the `allowed_failures` list. Unless you'd like that change as a separate PR?

Reply:
Sorry, I would've spared the last comment if I had seen https://github.com/bbatsov/rubocop/pull/4703#discussion_r139660502 before :smile:.

Yeah, should be passing now.

Reply:
@deivid-rodriguez Good catch. I wasn't really thinking when I made the change. I agree, since we support jRuby, we should enforce green builds. The build is passing in jRuby so I will remove it from the `allowed_failures` section.